modules/node/src/cfCore/cfCore.store.ts

```diff
@@ -1,4 +1,4 @@
-import { Injectable } from "@nestjs/common";
+import {Injectable} from '@nestjs/common';
```

Comment:
Yikes looks like linter wars. We have a prettier config for this, can you use that?